pkg/odo/cli/component/push.go

```diff
@@ -37,9 +37,20 @@ type PushOptions struct {
 	// devfile path
 	devfilePath string
 
+	// devfile commands
+	devfileBuildCommand string
+	devfileRunCommand   string
+
 	namespace string
 }
 
+// DevfileCommand encapsulates the build and run command
+// which can be explicitly provided during odo push
+type DevfileCommand struct {
```

Comment:
Is this needed? Don't believe it's used anywhere.

Reply:
nope, i forgot to remove it

Reply:
Removed in commit https://github.com/openshift/odo/commit/14d81a5d97bfc5672b4cb57797bd3d22ebb5deae